### ISSUE TITLE
Experiment details for the synthetic data

### ISSUE THREAD

**Issue by hammer-wang** [AMLab-Amsterdam/CEVAE#3]
Hi, 

I'm trying to replicate your results on the synthetic data experiment in section 4.2. I'd appreciate it a lot if you could help me with several questions. 

1. Which package did you use for implementing LR1 and LR2? Is it tensorflow or scikit-learning? If you used tensorflow, did you implement it as softmax regression (with two-dimensional output) or the standard logistic regression?

2. How did you evaluate the model? Did you do train-validation-test split? If so, what is the size of the test data?

3. Did you use early stopping when training the model? What was the stopping criterion you used? 

Thanks a lot! 

Best,
Haozhu

**Comment by clouizos** [comment]
Hi,

Apologies for the late reply, I just saw this issue. Let me answer your questions:

1. We used the scikit-learn implementation for logistic regression.

2. For this task we trained on the whole dataset. Notice that this is not an issue, as we care about the behaviour on the counterfactuals which are not part of the training dataset.

3.  We didn't use any stopping criterion but rather trained the model till convergence.

Hope this helps!
